Modules/Axis/Sources/Axis/String/Collection.swift

```diff
@@ -0,0 +1,16 @@
+extension Collection where Iterator.Element: Equatable, Iterator.Element == SubSequence.Iterator.Element {
+    func index(of part: Self) -> Index? {
+        guard count >= part.count else { return nil }
+
+        let _offset = stride(from: 0, through: count - part.count, by: 1).first {
```

Comment:
I like to use `first(where: {...})` (with the label and parenthesis) since it adds more clarity.